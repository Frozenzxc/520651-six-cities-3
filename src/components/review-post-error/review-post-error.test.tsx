import * as React from "react";
import * as renderer from "react-test-renderer";
import ReviewPostError from "./review-post-error";


it(`Should Review Post Error component render correctly`, () => {
  const tree = renderer
        .create(
            <ReviewPostError/>
        )
        .toJSON();

  expect(tree).toMatchSnapshot();
});
