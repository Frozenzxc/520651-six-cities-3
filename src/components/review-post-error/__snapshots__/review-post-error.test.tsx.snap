// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Review Post Error component render correctly 1`] = `
<div
  className="reviews__error"
>
  Попробуйте еще раз
</div>
`;
